src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/ReadyToRunCompilerContext.cs

```diff
@@ -43,6 +43,11 @@ public ReadyToRunCompilerContext(TargetDetails details, SharedGenericsMode gener
 
             // No architecture has completely stable handling of Vector<T> in the abi (Arm64 may change to SVE)
             _vectorOfTFieldLayoutAlgorithm = new VectorOfTFieldLayoutAlgorithm(_r2rFieldLayoutAlgorithm, _vectorFieldLayoutAlgorithm, matchingVectorType, bubbleIncludesCorelib);
+
+            if (oldTypeSystemContext != null)
+            {
+                base.InheritOpenModules(oldTypeSystemContext);
```

Comment:
```suggestion
                InheritOpenModules(oldTypeSystemContext);
```

We don't prefix with base unless its necessary to e.g. call a base method that is overriden.